homeworks/homework_1/task_2/head.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+from typing import List
+
+
+def print_head(file_names: List[str], rows_number: int = 10):
+    print_file_name = len(file_names) > 1
+    for file_name in file_names:
+        try:
+            with open(file_name) as f:
+                if print_file_name:
+                    print(f"==> {file_name} <==")
+                print("".join(list(f)[:rows_number]))
```

Comment:
Yeah, specifically here, let's also not load the entire file into memory just for the first 10 lines.